frontend/public/components/app.tsx

```diff
@@ -494,19 +492,25 @@ graphQLReady.onReady(() => {
     <Suspense fallback={<LoadingBox />}>
       <Provider store={store}>
         <ThemeProvider>
-          <AppInitSDK
-            configurations={{
-              appFetch: appInternalFetch,
-              apiDiscovery: initApiDiscovery,
-              initPlugins,
-            }}
-          >
-            <ToastProvider>
-              <PollConsoleUpdates />
-              <AdmissionWebhookWarningNotifications />
-              <AppRouter />
-            </ToastProvider>
-          </AppInitSDK>
+          <HelmetProvider>
+            <Helmet
+              titleTemplate={`%s · ${getBrandingDetails().productName}`}
+              defaultTitle={getBrandingDetails().productName}
```

Comment:
nit: we should store `productName` in a variable so we dont need to call `getBrandingDetails` twice

```
const { productName } = getBrandingDetails();
```